docker/nginx.conf

```diff
@@ -0,0 +1,34 @@
+events { worker_connections 1024; }
+
+http {
+    include    /etc/nginx/mime.types;
+    # include    /etc/nginx/proxy.conf;
+    # include    /etc/nginx/fastcgi.conf;    
+    
+    upstream girder {
+        server girder:8080;
+    }
+
+    client_max_body_size 100M;
+
+    server {
+
+        root /usr/share/nginx/html;
+
+        listen 80;
+
+        location /api/ {
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-NginX-Proxy true;
+            proxy_pass http://girder/api/;
```

Comment:
Is nginx also proxying girder? so I am wondering if we need to set the cors stuff.

Reply:
I just took the config that Brandon did. I think we still need cors stuff.